e2e/test/config/Configuration.IoTHub.cs

```diff
@@ -11,7 +11,7 @@ public static partial class Configuration
     {
         public static partial class IoTHub
         {
-            public static string ConnectionString => GetValue("IOTHUB_CONN_STRING_CSHARP");
+            public static string ConnectionString => GetValue("IOTHUB_CONNECTION_STRING");
```

Comment:
We use this in java already and there is no need for something different in .net. This will make the script cleaner for cross language resource creation.